packages/example/babel.config.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable import/no-commonjs */
+/* eslint-env node */
+
+module.exports = function(api) {
```

Comment:
There's supposed to be only one `babel.config.js` in a monorepo – in the root. But if it's working, then I guess it's ok. I'd try to change it to `.babelrc.js`

Reply:
Just kept what's generated by expo. Don't wanna mess with it 😅